libs/types/src/tokens.rs

```diff
@@ -212,6 +212,49 @@ pub struct CustomMetadata {
 	pub pool_currency: bool,
 }
 
+/// The Cross Chain Transferability property of an asset describes the way(s),
+/// if any, that said asset is cross-chain transferable. It may currently be
+/// transferable through Xcm, Centrifuge Connectors, or All .
+///
+/// NOTE: Once set to `All`, the asset is automatically transferable through any
+/// eventual new option added at a later stage. A migration might be required if
+/// that's undesirable for any registered asset.
+#[derive(
+	Clone,
+	Copy,
+	Default,
+	PartialOrd,
+	Ord,
+	PartialEq,
+	Eq,
+	Debug,
+	Encode,
+	Decode,
+	TypeInfo,
+	MaxEncodedLen,
+)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum CrossChainTransferability {
+	/// The asset is not transferable cross-chain
+	#[default]
+	None,
+
+	/// The asset is only transferable through XCM
+	Xcm(XcmMetadata),
+
+	/// The asset is only transferable through Centrifuge Connectors
+	Connectors,
+
+	/// The asset is transferable through all available options
+	All(XcmMetadata),
+}
+
+impl CrossChainTransferability {
+	pub fn includes_xcm(self) -> bool {
+		self != Self::None && matches!(self, Self::Xcm(..) | Self::All(..))
```

Comment:
Seems that the `matches!(self, Self::Xcm(..) | Self::All(..))` already covers the !Self::None case right? We might be able to simplify to: 
```suggestion
		matches!(self, Self::Xcm(..) | Self::All(..))
```

Reply:
Thank you 😇 the reason why I was checking that it wasn't None was because of legacy. We initially had the None variant and a function includes(other: CrosschainTransferability) but because now that we added None again but the XCM variant has a value, it wasn't ergonomic to have a function with that signature, so we chose for includes_xcm(self) and uses matches!; But with that I forgot that we no longer have to check for None. Thanks again ✌🏻